pokemon-minisite/src/app/app.component.ts

```diff
@@ -5,10 +5,11 @@ import { PokemonListComponent } from './components/pokemon-list/pokemon-list.com
 import { HeaderComponent } from './components/header/header.component';
 
 
+
 @Component({
```

Comment:
you don't need the imports here. The imports needs to be in the module of the component